k8s/aat/common/ia/home-office-integration-api.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-home-office-integration-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
```

Comment:
The new annotations should be used for this, search fluxcd in the repo

Reply:
I've changed ... not sure if I needed others I saw on `draft-store` like: ```    repository.fluxcd.io/smokecron: java.smoketestscron.image
    filter.fluxcd.io/smokecron: glob:prod-*
    repository.fluxcd.io/functionalcron: java.functionaltestscron.image
    filter.fluxcd.io/functionalcron: glob:prod-*
    repository.fluxcd.io/smoke: java.smoketests.image
    filter.fluxcd.io/smoke: glob:prod-*
    repository.fluxcd.io/functional: java.functionaltests.image
    filter.fluxcd.io/functional: glob:prod-*```